civihr_default_theme/scss/partials/_menu.scss

```diff
@@ -62,3 +62,98 @@
         }
     }
 }
+
+.chr_header__settings-menu {
+  position: relative;
+  display: inline-block;
+  width: 50px;
+  height: 50px;
+  text-align: center;
+  float: right;
+
+  &:hover {
+    background-color: $crm_gray-darker-alt;
+
+    > .chr_header__settings-sub-menu {
+      display: block;
+    }
+  }
+
+  .chr_header__settings-sub-menu {
+    background: $crm_gray-darker-alt;
+    display: none;
+    position: absolute;
+    right: 0;
+    z-index: 99;
+
+    li {
+      text-align: left;
+    }
+
+    a {
+      color: $light-pale;
+      font-size: 1.3rem;
+      display: block;
+      padding: 0 30px 0 20px;
+      white-space: nowrap;
+
+      &:hover {
+        background: $crm_brand-secondary;
+        color: $light;
+      }
+
+      > i {
+        margin-right: 10px;
+      }
+    }
+  }
+}
+
+.chr_header__home-menu {
+
+  .chr_header__home-sub-menu {
```

Comment:
I'm not talking about the names. I'm talking about nesting selectors, and I believe it should be done only when necessary, not for the sake of mimicking the DOM